src/engine/sparqlExpressions/UuidExpressions.h

```diff
@@ -0,0 +1,73 @@
+//  Copyright 2024, University of Freiburg,
+//                  Chair of Algorithms and Data Structures
+//  Author: Hannes Baumann <baumannh@informatik.uni-freiburg.de>
+
+// Test for UuidExpression can be found in RandomExpressionTest.cpp
+
+#pragma once
+
+#include "engine/sparqlExpressions/SparqlExpression.h"
+#include "util/ChunkedForLoop.h"
+#include "util/Random.h"
+
+namespace sparqlExpression {
+
+using LiteralOrIri = ad_utility::triple_component::LiteralOrIri;
+
+inline constexpr auto fromLiteral = [](std::string_view str) {
```

Comment:
1. `inline` is redundant for `constexpr` values, so it can be dropped.
2. Those inline functions are a bit prone to multiple definition errors etc, better move them into a namespace  `sparqlExpression::uuidExpression::detail` or something.

Reply:
Somehow the tests for `builtInCall` don't pass if i remove the `inline`.